tests/testthat/test-alpha.r

```diff
@@ -29,3 +29,22 @@ test_that("col values recycled to match alpha", {
 
   expect_equal(alphas, reds_alpha)
 })
+
+test_that("preserves element names", {
+  expect_equal(c("first", "second", "third"), names(alpha(c(first   = "gold", second   = "lightgray", third   = "#cd7f32"), .5)))   # not quoted
```

Comment:
I think you only need a single expectation here; otherwise you're acutally testing the behaviour of `names<-`.

Reply:
@hadley, I'm leaving the two cases where the first/last element is unnamed.  It's a habit of mine protecting against one-off errors.  If you feel strongly, I'll thin it out more.